src/Surfnet/StepupMiddleware/CommandHandlingBundle/Tests/Configuration/CommandHandler/InstitutionConfigurationCommandHandlerTest.php

```diff
@@ -220,6 +251,98 @@ public function show_raa_contact_information_option_is_changed_if_its_given_valu
      * @test
      * @group command-handler
      */
+    public function allowed_second_factor_list_is_changed_if_its_values_are_different_than_the_current_list()
+    {
+        $institution                     = new Institution('Institution');
+        $institutionConfigurationId      = InstitutionConfigurationId::normalizedFrom($institution);
+        $useRaLocationsOption            = UseRaLocationsOption::getDefault();
+        $showRaaContactInformationOption = ShowRaaContactInformationOption::getDefault();
+        $originalAllowedSecondFactorList = AllowedSecondFactorList::blank();
+
+        $secondFactorsToAllow = ['sms', 'yubikey'];
+
+        $allowedSecondFactors = array_map(function ($secondFactorType) {
+            return new SecondFactorType($secondFactorType);
+        }, $secondFactorsToAllow);
+        $allowedSecondFactorList = AllowedSecondFactorList::ofTypes($allowedSecondFactors);
```

Comment:
I did it this way, because the `$secondFactorsToAllow` variable is still used in the `$command->allowedSecondFactors` and I like them to point to the same value internally as this reduces bugs in testing code.
I will change the `array_map` to a mapping by hand by using array access. This keeps the advantage of both.